### DISCUSSION TITLE
Links are not working for vocab files

### DISCUSSION THREAD

**Discussion by ShivamPR21** [stella-cv/stella_vslam#60] [Q&A] [answered]
Hi! I was trying to run the SLAM with standard dataset according to [instructions provided on the docs page](https://openvslam-community.readthedocs.io/en/latest/example.html#section-example-standard-datasets), but all the links referring to the `vocab` files for `DBoW2` aren't working.
![image](https://user-images.githubusercontent.com/46261813/111144397-097dae00-85ad-11eb-89fb-0d08cf53dcf6.png)

**Comment by ymd-stella** [comment, marked as answer]
Thanks for the report. It can be found at https://github.com/OpenVSLAM-Community/DBoW2_orb_vocab.

**Reply by ymd-stella** [reply]
Fixed in 7b0b81c463a0b823e3674e9c23e7609496ce8bf8.

**Reply by ShivamPR21** [reply]
Thanks, for the fix.
